include/deal.II/fe/fe_poly.h

```diff
@@ -278,10 +278,38 @@ class FE_Poly : public FiniteElement<dim,spacedim>
      * then only have to apply the transformation (which is a matrix-vector
      * multiplication) when visiting an actual cell.
      */
+
     std::vector<std::vector<Tensor<1,dim> > > shape_gradients;
+    /**
```

Comment:
actually, while you're at it, also remove the additional empty line between the comment and the declaration. You can also add an empty line between the declaration and the comment below (which refers to the next variable).